src/lib.rs

```diff
@@ -10,6 +10,7 @@
 // std is needed to run tests, but otherwise we don't need it.
 #![cfg_attr(not(test), no_std)]
 #![warn(missing_docs)]
+#![allow(clippy::let_unit_value)]
```

Comment:
does this need to be top-level?

Reply:
Hmm, actually it looks like it's not even needed at all anymore.